package.json

```diff
@@ -45,6 +47,12 @@
     "ts-jest": "^29.1.1",
     "typescript": "^5.1.6"
   },
+  "optionalDependencies": {
+    "@nx/nx-darwin-arm64": "16.5.1",
+    "@nx/nx-darwin-x64": "16.5.1",
+    "@nx/nx-linux-x64-gnu": "16.5.1",
+    "@nx/nx-win32-x64-msvc": "16.5.1"
+  },
```

Comment:
Context about this optional deps here: https://github.com/nrwl/nx/issues/17767. NX peer dep causes issues with NPM during CI process.